tools/portforward/tunnel.go

```diff
@@ -34,7 +34,9 @@ type Tunnel struct {
 	Local     int
 	Remote    int
 	Namespace string
-	PodName   string
+	Resource  string
+	Name      string
+	PodName   string // Deprecated: Use "Name" field instead.
```

Comment:
Don't deprecate, just rename the field. We can fix where build breaks.

Reply:
done